moduleframework/common.py

```diff
@@ -231,3 +192,253 @@ def sanitize_cmd(cmd):
         if char in cmd:
             cmd = cmd.replace(char, '\\'.join(char))
     return cmd
+
+
+def get_profile():
+    """
+    Return profile name string
+
+    :return: str
+    """
+    amodule = os.environ.get('PROFILE')
```

Comment:
why 'amodule' and 'return amodule' and not 'profile' if a function called 'get_profile'?